pkg/infrastructure/kogito_service.go

```diff
@@ -80,6 +85,18 @@ func injectURLIntoKogitoApps(client *client.Client, namespace string, serviceHTT
 	return nil
 }
 
+// GetKogitoServiceEndpoint gets the endpoint depending on
+// if the envVarKogitoServiceURL is set (for when running
+// operator locally). Else, the internal endpoint is
+// returned.
+func GetKogitoServiceEndpoint(kogitoService v1alpha1.KogitoService) string {
+	externalURL := os.Getenv(envVarKogitoServiceURL)
+	if len(externalURL) > 0 {
+		return externalURL
+	}
+	return CreateKogitoServiceURI(kogitoService)
```

Comment:
Please review the source code for calls to `CreateKogitoServiceURI` and let's wrap everyone around that and make it private.